### ISSUE TITLE
A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader : No proper role and expand/collapse state is announced for the "New" control for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/azuredatastudio#11544]
**"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-manai%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”**
GitHubTags:#A11y_AzureDataStudioJuly2020;#A11y_July27_2020_TestPass;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#MAC;#Screenreader;#VoiceOver;#A11ySev1;#Benchmark;#MAS1.3.1;#MAS4.1.2;#MAS4.2.1#FTP;

### Environment Details:

Application Name: Azure Data Studio
Application Version: 1.21.0-insider
Commit: eccf3cf
Date: 2020-07-24T09:28:31.172Z
VS Code: 1.48.0
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating system: macOS Catalina (Version 10.15.6 (19G73)
Screen Reader: VoiceOver
MAS References: MAS1.3.1, MAS4.1.2,MAS4.2.1

### Repro Steps:

1. Launch Azure Data Studio Insiders application.
2. If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3. Start screen reader, navigate to "New' control and listen if the role and expand/collapse state of the control is announced or not.

### Actual:

When screen reader users navigate to the "New" control it is announced as 'empty' navigation, no proper role is announced for this control and on accessing the control some content gets expanded and collapsed but the expand/collapse state is not announced for the screen reader users.

### Expected:

For screen reader users the "New' control should be announced with proper role as 'button' and the expand/collapse should also be announced for the control on accessing the control.

### User Impact:

If proper role and information of the state of an interactive element is not conveyed to the screen reader users for the changes appearing on the screen then they will miss out on the information that is available for the other users.

### Attachment link for Reference:
[11544_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader No proper role and expand:collapse state is announced for the "New" control for screen reader users.zip](https://github.com/microsoft/azuredatastudio/files/4985411/11544_A11y_AzureDataStudio_Welcome_AzureDataStudio-Insiders_ScreenReader.No.proper.role.and.expand.collapse.state.is.announced.for.the.New.control.for.screen.reader.users.zip)

**Comment by Raghu-addepalli** [comment]
https://user-images.githubusercontent.com/91586743/220330460-2e318cab-3f92-4f75-af42-524f72830390.mov

verified the issue Now it is not reproducing and added attachment.
Githubtags:  #Closed;
